site/src/helpers/styling.ts

```diff
@@ -0,0 +1,38 @@
+import { type Theme } from '@nivo/core';
+
+export const tooltipStyle: Theme = {
+  tooltip: {
+    container: {
+      background: 'rgba(0,0,0,.87)',
+      color: '#ffffff',
+      fontSize: '1.2rem',
+      outline: 'none',
+      margin: 0,
+      padding: '0.25em 0.5em',
+      borderRadius: '2px',
+    },
+  },
+};
+
+export const getTheme = (darkMode: boolean): Theme => {
+  return {
+    axis: {
+      ticks: {
+        text: {
+          fill: darkMode ? '#eee' : '#333',
+        },
+      },
+      legend: {
+        text: {
+          fill: darkMode ? '#eee' : '#333',
+        },
+      },
+    },
+  };
+};
+
+export const getCssVariable = (name: string): string => {
+  const bodyStyles = getComputedStyle(document.body);
+  const valueFromBody = bodyStyles.getPropertyValue(name).trim();
+  return valueFromBody || '#ffffff';
```

Comment:
This should mirror the behavior of getting a CSS variable, as the function name states. I believe it would be more correct to return an empty string instead of defaulting to white.

If you need a default value for special cases, you should instead take an optional second parameter `(..., defaultValue: string = '')` and return `valueFromBody || defaultValue`